internal/services/v0/developer_test.go

```diff
@@ -98,11 +98,12 @@ func TestEditCheck(t *testing.T) {
 			[]*v0.RelationTuple{},
 			[]*v0.RelationTuple{},
 			&v0.DeveloperError{
-				Message: "parse error in `schema`, line 3, column 4: Expected identifier, found token TokenTypeRightBrace",
+				Message: "Expected identifier, found token TokenTypeRightBrace",
```

Comment:
would this message be more readable with `}` instead of `TokenTypeRightBrace`?

Reply:
That's actually what I'm passing along now out of band from the error message itself. We could change the error message as well, but it would require a reverse map of the token types (which, I suppose would be fairly easy to do), but may appear odd for non-ASCII runes

Reply:
This could be a follow up, but it would it make sense to not keep a reverse map, but instead keep an "error message map"? then we can write the best representation for any given token.

Reply:
Yeah, we could do that too

Reply:
Let's do that as a followup